workspace-manager-clienttests/render-k8s-config.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This script renders the k8s service account secrets for namespace access
+# NAME_SPACE: ichang, zloery, or preview namespace etc.
+NAMESPACE=${1}
+VAULT_TOKEN=${2:-$(cat "$HOME"/.vault-token)}
+TESTRUNNER_K8S_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/integration/${NAMESPACE}/testrunner-k8s-sa
+TESTRUNNER_K8S_SERVICE_ACCOUNT_CA_OUTPUT_PATH=$(dirname "$0")/rendered/testrunner-k8s-sa-client-key-data.crt
+TESTRUNNER_K8S_SERVICE_ACCOUNT_TOKEN_OUTPUT_PATH=$(dirname "$0")/rendered/testrunner-k8s-sa-token
+
+DSDE_TOOLBOX_DOCKER_IMAGE=broadinstitute/dsde-toolbox:consul-0.20.0
+
+docker run --rm -e VAULT_TOKEN="$VAULT_TOKEN" $DSDE_TOOLBOX_DOCKER_IMAGE \
+    vault read -format=json ${TESTRUNNER_K8S_SERVICE_ACCOUNT_VAULT_PATH} | \
+    jq -r .data.key | base64 --decode | \
+    jq -r ".data[\"ca.crt\"]" > ${TESTRUNNER_K8S_SERVICE_ACCOUNT_CA_OUTPUT_PATH}
+
+docker run --rm -e VAULT_TOKEN="$VAULT_TOKEN" $DSDE_TOOLBOX_DOCKER_IMAGE \
+    vault read -format=json ${TESTRUNNER_K8S_SERVICE_ACCOUNT_VAULT_PATH} | \
+    jq -r .data.key | base64 --decode | \
+    jq -r .data.token | base64 --decode > ${TESTRUNNER_K8S_SERVICE_ACCOUNT_TOKEN_OUTPUT_PATH}
```

Comment:
nit: add newline at end of file.

Why is `render-k8s-config.sh` separate from `render-config.sh`? Is it because resiliency tests will be run less commonly, so we don't want the k8s credentials always rendered? If  that's not a concern, it might be nice to combine the two scripts for ease of use.

Reply:
Per `devOps` suggestion, we intentionally separate tests that won't need k8s credentials for manipulating Kubernetes resources (regular perf, integration tests) from resiliency tests that require the special RBAC access. The Test Runner Library has been enhanced to reinforce this separation of control.